exporter/collector/config.go

```diff
@@ -35,6 +35,8 @@ import (
 
 const (
 	DefaultTimeout = 12 * time.Second // Consistent with Cloud Monitoring's timeout
+
+	CompressionGzip = "gzip"
```

Comment:
nit: does this need to be exported?

Reply:
also seems like you use `gzip.Name` as a constant elsewhere.  Might be good to use that instead.